### ISSUE TITLE
Explanation of metadata keys in ontology terms for key 1

### ISSUE THREAD

**Issue by chrwm** [OpenEnergyPlatform/oemetadata#58]
Metadata keys must be matched with ontology terms to address `key #1` `@context`.
The table below is a first proposal.

| # | key | ontology_id
-- | -- | --
1 | `@context` |  
2 | name |  
3 | title |  
4 | id |  
5 | `@id` |  
6 | description |  
7 | language |  
8 | subject |  
9 | keywords |  
10 | publicationDate |  
11 | context |  
11.1 | homepage |  
11.2 | documentation |  
11.3 | sourceCode |  
11.4 | contact |  
11.5 | grantNo |  
11.6 | fundingAgency |  
11.7 | fundingAgencyLogo |  
11.8 | publisherLogo |  
12 | spatial |  
12.1 | location |  
12.2 | extent |  
12.3 | resolution |  
13 | temporal |  
13.1 | referenceDate |  
13.2 | timeseries |  
13.2.1 | start |  
13.2.2 | end |  
13.2.3 | resolution |  
13.2.4 | alignment |  
13.2.5 | aggregationType |  
14 | sources |  
14.1 | title |  
14.2 | description |  
14.3 | path |  
14.4 | licenses |  
14.4.1 | name |  
14.4.2 | title |  
14.4.3 | path |  
14.4.4 | instruction |  
14.4.5 | attribution |  
15 | licenses |  
15.1 | name |  
15.2 | title |  
15.3 | path |  
15.4 | instruction |  
15.5 | attribution |  
16 | contributors |  
16.1 | title |  
16.2 | email |  
16.3 | date |  
16.4 | object |  
16.5 | comment |  
17 | resources |  
17.1 | profile |  
17.2 | name |  
17.3 | path |  
17.4 | format |  
17.5 | encoding |  
17.6 | schema |  
17.6.1 | fields |  
17.6.1.1 | name |  
17.6.1.2 | description |  
17.6.1.3 | type |  
17.6.1.4 | is_about | obo:IAO_0000136
17.6.1.5 | value_reference |  
17.6.1.6 | unit | oeo:OEO_00040010
17.6.2 | primaryKey |  
17.6.3 | foreignKeys |  
17.6.3.1 | fields |  
17.6.3.2 | reference |  
17.6.3.2.1 | resource |  
17.6.3.2.2 | fields |  
17.7 | dialect |  
17.7.1 | delimiter |  
17.7.2 | decimalSeparator |  
18 | review |  
18.1 | path |  
18.2 | badge |  
19 | metaMetadata |  
19.1 | metadataVersion |  
19.2 | metadataLicense |  
19.2.1 | name |  
19.2.2 | title |  
19.2.3 | path |  
20 | _comment |  
20.1 | metadata |  
20.2 | dates |  
20.3 | units | oeo:OEO_00040010 
20.4 | languages |  
20.5 | licenses |  
20.6 | review |

**Comment by adelmemariani** [comment]
Some terms are not covered by the OEO yet. In metadata V1.4, the [DCAT](https://www.w3.org/TR/vocab-dcat/#introduction) is used for these terms. I will use the OEO as much as possible.

**Comment by stap-m** [comment]
@adelmemariani can you collect the missing terms and open an issue in the [OEO repo](https://github.com/OpenEnergyPlatform/ontology) please? We should add the missing terms soon.

**Comment by chrwm** [comment]
Hej @adelmemariani, I've updated the table above with suggestions to apply the OEO as best as possible to the metadata (I noticed I still had them locally).
However, I was unsure whether to only describe the metadata keys with classes or if object properties are also allowed in my attempt. In doubt, I linked both if they seemed related.

**Comment by chrwm** [comment]
I'll continue and start the implementation of missing OEO concepts for oemetadata keys in the OEO repo.

**Comment by jh-RLI** [comment]
I'm not sure if this is related, but I want to mention it: [OMI](https://github.com/OpenEnergyPlatform/omi/tree/dev/src/omi/dialects/rdf) has already implemented some functionality related to RDF based on the oemetadata (v1.4.1 so far). There is a example [.ttl file](https://github.com/OpenEnergyPlatform/omi/blob/dev/tests/data/metadata_v14.ttl).

**Comment by chrwm** [comment]
Yes thanks, [.ttl file](https://github.com/OpenEnergyPlatform/omi/blob/dev/tests/data/metadata_v14.ttl) is relevant. I'll review this before I proceed.

**Comment by Ludee** [comment]
I just found this 
https://nfdi4ing.pages.rwth-aachen.de/metadata4ing/metadata4ing/
